docs/v3.x/migration-guide/migration-guide-3.3.x-to-3.4.0.md

```diff
@@ -0,0 +1,17 @@
+# Migration guide from 3.3.x to 3.4.0
+
+**Make sure your server is not running until the end of the migration**
+
+:::warning
+If you are using **extensions** to create custom code or modifying existing code, you will need to update your code and compare your version to the new changes on the repository.
+<br>
+Not updating your **extensions** can break your app in unexpected ways that we cannot predict.
+:::
+
+## Migration
+
+1. First, update your app as usual by following the basic [version update guide](../guides/update-version.md) and then come back here
+2. Download this script: <a :href="$withBase('/assets/migrations/scripts/migrate-3.4.0.js')" download>migrate-3.4.0.js</a>
```

Comment:
Would it be better to link to the GitHub script instead or do you think it's better to include this in the docs?

Reply:
The script will not be in the main repo strapi/strapi because it has no sense for future users, but I'm not against putting it in another repo if there is a good one. For the moment it's where we thought it would be the best.
Ping @alexandrebodin

Reply:
Ah that's fine, I don't mind if it's in the doc repo just wasn't sure if it would "stay" in the monorepo.